src/Compilers/Core/Portable/StrongName/CryptoBlobParser.cs

```diff
@@ -171,7 +171,13 @@ internal static bool IsValidPublicKey(ImmutableArray<byte> blob)
             // re-add padding for exponent
             w.WriteUInt32(pubExp);
 
-            w.WriteBytes(pubKeyData);
+            unsafe
+            {
+                fixed (byte* bytes = pubKeyData)
+                {
+                    w.WriteBytes(bytes, pubKeyData.Length);
```

Comment:
`BlobWriter.WriteSpan` seems like one of the new APIs we should be tracking in the linked issue.